tools/db_crashtest.py

```diff
@@ -880,6 +889,8 @@ def finalize_and_sanitize(src_params):
         dest_params["use_multi_cf_iterator"] = 0
         # only works with write committed policy
         dest_params["commit_bypass_memtable_one_in"] = 0
+        # not compatible with Remote Compaction yet
+        dest_params["remote_compaction_worker_threads"] = 0
```

Comment:
Can we also sanity check relevant options before configuring CompactionService in db_stress_test_base.cc?